airflow/config_templates/config.yml

```
@@ -1080,6 +1080,13 @@
       type: string
       example: ~
       default: "30"
+    - name: site_title
+      description: |
+        Sets a title to show in the browser tab and the DAGs overview page
+      version_added: ~
+      type: string
+      example: ~
+      default: "DAGs"
```

```suggestion
      default: 
```
We will have more freedom if we do not define a default value. The default value can be specified while reading.